ext/http/lib.rs

```diff
@@ -231,6 +245,265 @@ impl From<tokio::net::unix::SocketAddr> for HttpSocketAddr {
+      Version::HTTP_11 => "1.1",
+      Version::HTTP_2 => "2.0",
+      Version::HTTP_3 => "3.0",
+      _ => unreachable!(),
+    }
+  }
+
+  fn for_counter(&self) -> Vec<deno_telemetry::KeyValue> {
+    let mut attributes = vec![
+      deno_telemetry::KeyValue::new(
+        "http.request.method",
+        self.http_request_method.clone(),
+      ),
+      deno_telemetry::KeyValue::new("url.scheme", self.url_scheme.clone()),
+    ];
+
+    if let Some(address) = self.server_address.clone() {
+      attributes.push(deno_telemetry::KeyValue::new("server.address", address));
+    }
+    if let Some(port) = self.server_port {
+      attributes.push(deno_telemetry::KeyValue::new("server.port", port));
+    }
+
+    attributes
+  }
+
+  fn for_histogram(&self) -> Vec<deno_telemetry::KeyValue> {
+    let mut histogram_attributes = vec![
+      deno_telemetry::KeyValue::new(
+        "http.request.method",
+        self.http_request_method.clone(),
+      ),
+      deno_telemetry::KeyValue::new("url.scheme", self.url_scheme.clone()),
+      deno_telemetry::KeyValue::new(
+        "network.protocol.version",
+        self.network_protocol_version,
+      ),
+    ];
+
+    if let Some(address) = self.server_address.clone() {
+      histogram_attributes
+        .push(deno_telemetry::KeyValue::new("server.address", address));
+    }
+    if let Some(port) = self.server_port {
+      histogram_attributes
+        .push(deno_telemetry::KeyValue::new("server.port", port));
+    }
+    if let Some(status_code) = self.http_response_status_code {
+      histogram_attributes.push(deno_telemetry::KeyValue::new(
+        "http.response.status_code",
+        status_code,
+      ));
+    }
+
+    if let Some(error) = self.error_type {
+      histogram_attributes
+        .push(deno_telemetry::KeyValue::new("error.type", error));
+    }
+
+    histogram_attributes
+  }
+}
+
+impl OtelInfo {
+  fn new(
+    instant: std::time::Instant,
+    request_size: u64,
+    attributes: OtelInfoAttributes,
+  ) -> Self {
+    let otel = OTEL_GLOBALS.get().unwrap();
+    let collectors = OTEL_COLLECTORS.get_or_init(|| {
+      let meter = otel
+        .meter_provider
+        .meter_with_scope(otel.builtin_instrumentation_scope.clone());
+
+      let duration = meter
+        .f64_histogram("http.server.request.duration")
+        .with_unit("s")
+        .with_description("Duration of HTTP server requests.")
+        .with_boundaries(vec![
+          0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1.0, 2.5, 5.0,
+          7.5, 10.0,
+        ])
+        .build();
+
+      let active_requests = meter
+        .i64_up_down_counter("http.server.active_requests")
+        .with_unit("{request}")
+        .with_description("Number of active HTTP server requests.")
+        .build();
+
+      let request_size = meter
+        .u64_histogram("http.server.request.body.size")
+        .with_unit("By")
+        .with_description("Size of HTTP server request bodies.")
```

Comment:
Can you make the buckets for this `[0, 100, 1000, 10000, 100000, 1000000, 10000000, 100000000, 1000000000]` (also in the other size histogram).